### ISSUE TITLE
TeX/fontdata.js instead of STIX

### ISSUE THREAD

**Issue by grinapo** [xwiki-labs/cryptpad#519]
[This is a fresh install with lot of uncanny tricks so it may be misconfiguration as well. In that case please kindly kick my butt.]

**Describe the bug**
```
2020/03/28 00:18:45 [error] 4410#4410: *270171 open() "/home/cryptpad/cryptpad/customize.dist//pad/mathjax/jax/output/HTML-CSS/fonts/TeX/fontdata.js" failed (2: No such file or directory), client: 10.7
```
it seems to be rather here:
```
./www/pad/mathjax/jax/output/HTML-CSS/fonts/STIX/fontdata.js
```

**Where did it happen?**
My own instance, https://cryptpad.grin.hu/

**To Reproduce**
Probably in rich text. I observed that I can't sustain a font change and it changes back to (that ugly) Arial all the time. Started looking for errors, and that's what I found.

**Browser (please complete the following information):**
 - OS: Debian/Linux
 - Browser: Chromium 80.0.3987.149
 - Extensions installed: it's in private window, none.
 - Browser tweaks: chrome incognito mode

**Comment by ansuz** [comment]
Could you describe the bug in more detail? A log statement and a reference to a line number are not enough to convey what's going wrong.

**Comment by grinapo** [comment]
I'm not sure there's a lot of more details.
The local installation includes a huge pile of javascript from all over the place. It is probably preprocessed, and the final paths are generated. 
It seems that some pages include `fontdata.js` from the wrong place:

- it is at `./www/pad/mathjax/jax/output/HTML-CSS/fonts/STIX/` in the filesystem

while the html tries to include from the directory

-  `/home/cryptpad/cryptpad/customize.dist//pad/mathjax/jax/output/HTML-CSS/fonts/TeX/`

which is basically same as

- `./www/pad/mathjax/jax/output/HTML-CSS/fonts/STIX/`

So the webpage includes the file from the `TeX` directory while the file is in the `STIX` directory. Of if there should be a `TeX` directory with the file in it then it isn't there.

**Comment by ansuz** [comment]
I believe I was able to reproduce the issue on your instance. I used Firefox since i had it open at the time, suggesting that it's not an issue with Chrome or an add-on.

The steps I followed:

1. Load a rich text pad
2. open the network tab of the browser's console
3. click the "mathjax" button which opens a dialog containing a prompt for some _TeX_ code
4. observe that the network tab displays a 404 warning for `fontdata.js`

I followed the same steps on https://cryptpad.fr and on my personal instance (which uses the provided example nginx config) and was not able to reproduce, suggesting that the problem stems from your configuration.
